omp/__init__.py

```diff
@@ -44,28 +44,8 @@ def init_msvc(self, ver):
             self.version = 20
 
     def get_libomp_names(self):
-        """Return list of OpenMP libraries to try, based on platform and
-        compiler detected."""
-        if cxx is None:
-            # Can't tell what compiler we're using, guessing we need libgomp
-            names = ['libgomp']
```

Comment:
You could keep the changes here (assuming they had some reason in the first place) and just strip the `lib`...?

Reply:
No, that's overly complex for not much gain.